Microsoft.Toolkit.Uwp.UI/ImageCache/ImageCache.cs

```diff
@@ -156,7 +176,7 @@ public static int MaxMemoryCacheSize
             catch (Exception ex)
             {
                 System.Diagnostics.Debug.WriteLine(ex.Message);
-                return null;
+                throw ex;
```

Comment:
I'm still not convinced this needs to throw an exception. Where this method is called you catch the exception and return null. Well if you don't throw the exception here, this method would just return null, so the calling method would still just return null. Make sense?